frontend/packages/kubevirt-plugin/src/statuses/vm/vm.ts

```diff
@@ -230,22 +235,27 @@ export const getVMStatus = ({
   pods,
   migrations,
 }: {
-  vm: VMKind;
+  vm?: VMKind;
   vmi?: VMIKind;
   pods?: PodKind[];
   migrations?: K8sResourceKind[];
 }): VMStatus => {
-  const launcherPod = findVMPod(vm, pods);
+  const vmLike = vm || vmi;
+  const launcherPod = findVMPod(vmLike, pods);
   return (
-    isV2VConversion(vm, pods) || // these statuses must precede isRunning() because they do not rely on ready vms
-    isBeingMigrated(vm, migrations) || //  -||-
-    isBeingImported(vm, pods) || //  -||-
-    isBeingStopped(vm, launcherPod) ||
-    isRunning(vm) ||
+    isV2VConversion(vmLike, pods) || // these statuses must precede isRunning() because they do not rely on ready vms
+    isBeingMigrated(vmLike, migrations) || //  -||-
+    (vm && isBeingImported(vm, pods)) || //  -||-
+    (vmi && isBeingStopped(vm, launcherPod)) ||
```

Comment:
the vmi check should go inside so we check for the status of the launcher pod

Reply:
done